miniserver/.env.example

```diff
@@ -46,7 +46,7 @@ POLLING_INTERVAL=1000
 DEFAULT_NETWORK='eth-local'
 GAS_LIMIT='67219000'
 #GAS_PRICE='10000000'
-GAS_PRICE='413290302'
+GAS_PRICE='529942254'
```

Comment:
should use type 2 transactions instead of arbitrary numbers, or manually set base fee in emulating local eth blockchain

Reply:
Good Point 🙏 
An example of how to `use type 2 transactions` would be useful
Initial google shows these EIPS
* [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md): A transaction pricing mechanism that includes fixed-per-block network fee that is burned and dynamically expands/contracts block sizes to deal with transient congestion.
* [EIP-2178](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2718.md): TransactionType || TransactionPayload is a valid transaction and TransactionType || ReceiptPayload is a valid transaction receipt where TransactionType identifies the format of the transaction and *Payload is the transaction/receipt contents, which are defined in future EIPs.

Will investigate further.

Reply:
For now I've updated `prepareExecute` to retrieve the `gasPrice` 
`const gasPrice = await rpc.gasPrice({ network: config.defaultNetwork })`
This introduces an additional rpc call on each transaction
I could note see a way to drive a type 2 transaction in [ethers](https://docs.ethers.io/v5/api/contract/contract/#Contract--write) 

@polymorpher let me know your thoughts on this and whether the additional rpc call is to burdensome.

Reply:
that's fine, type 2 is not supported on all chains, including ganache